frontend/src/component/common/Table/cells/FeatureSeenCell/LastSeenProgress/LastSeenProgress.tsx

```diff
@@ -0,0 +1,52 @@
+import { styled, CircularProgress, Box } from '@mui/material';
+
+const ProgressContainer = styled('div')(({ theme }) => ({
+    display: 'flex',
+    flexDirection: 'row',
+    gap: theme.spacing(1),
+    width: '20%',
+    justifyContent: 'flex-end',
+}));
+
+const BackgroundCircularProgress = styled(CircularProgress)(({ theme }) => ({
+    color: theme.palette.divider,
+}));
+
+const MainCircularProgress = styled(CircularProgress)(({ theme }) => ({
+    color: theme.palette.primary.main,
+    position: 'absolute',
+    left: 0,
+}));
+
+interface ILastSeenProgressProps {
+    yes: number | undefined;
+    no: number | undefined;
+}
+
+export const LastSeenProgress = ({ yes, no }: ILastSeenProgressProps) => {
+    const noData = yes === undefined || no === undefined || yes + no === 0;
+    if (noData) {
+        return <Box />;
```

Comment:
why not null?

Reply:
This component is used in a lot of places in UI. 

1. Project screen
2. Project health screen
3. Archive table
and more and more.

Only the project screen has this data in payload(coming from search endpoint), otherwise it is undefined.